certvalidator/validate.py

```diff
@@ -1447,7 +1454,10 @@ def verify_crl(cert, path, validation_context, use_deltas=True, cert_description
                 certificate_list
             ))
             continue
-        if moment > certificate_list['tbs_cert_list']['next_update'].native:
+
+        # according to RFC 5280 5.1.2.5 next_update is optional
+        if certificate_list['tbs_cert_list']['next_update'] and \
+           moment > certificate_list['tbs_cert_list']['next_update'].native:
```

Comment:
```
   Conforming CRL issuers MUST include the nextUpdate field in all CRLs.
   Note that the ASN.1 syntax of TBSCertList describes this field as
   OPTIONAL, which is consistent with the ASN.1 structure defined in
   [X.509].  The behavior of clients processing CRLs that omit
   nextUpdate is not specified by this profile.
```
So you are completely correct here regarding RFC 5280. I was misinterpreting it based on some a non conforming CRL. There does appear to be some expectation of non-conforming CRLs which omit the field though... I'll have to take a look at hard/soft fail modes.